.changeset/olive-forks-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"@twilio-paste/progress-bar": patch
+---
+
+[Progress bar] Improve TsDoc type annotations
```

Comment:
question: what changes does this log refer to?

Reply:
update: closing out this PR because shadi is out today and removing this file because the tsdoc type annotation improvements happen in the i-p types PR (#3557) which already has this changeset.